### PR TITLE
Add new OT station

### PR DESCRIPTION
**Pull request by kannappansomasundaram1**
Add new OT station Corato sud ospedale

### CODE REVIEW

**Review by tl-johnw — APPROVED** [review]

**Comment by julien-duponchelle** [comment]
If I'm correct you need to add 29255 as parent https://trainline-eu.github.io/stations-studio/#/station/29255

File: stations.csv [hunk 1]
```diff
@@ -64047,3 +64047,4 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 68188;Le Bouscat Sainte-Germaine;le-bouscat-sainte-germaine;;87738096;44.87090;-0.61400;;FR;Europe/Paris;f;f;f;t;t;f;FRKTO;;t;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68188;
 68189;Andria Sud;andria-sud;8355025;;41.224651;16.314623;29245;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;8355025;t;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68189;
 68190;Ostarije;ostarije;7875111;;45.234444;15.296389;;HR;Europe/Zagreb;t;f;f;t;t;f;;;f;;f;7875111;f;;f;;f;;f;;f;;f;7875111;t;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68190;
+68191;Corato sud ospedale;Corato-sud-ospedale;;;41.14550687319366;16.419894058269794;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;;urn:trainline:public:nloc:csv68191;
```
Comment (misdoro) [inline stations.csv:64050]: Should the slug be all lower-case?

**Review by misdoro — CHANGES_REQUESTED** [review]

File: stations.csv (hunk 1 above)
Comment (kannappansomasundaram1) [reply stations.csv:64050]: I will change that

Comment (misdoro) [reply stations.csv:64050]: Thank you

**Review by misdoro — APPROVED** [review]

**Pull request merged**